freud/order/RotationalAutocorrelation.cc

```diff
@@ -5,6 +5,7 @@
 
 #include "utils.h"
 #include <cmath>
+// quat imported from VectorMath in rotationalautocorrelation.h
```

Comment:
```suggestion
#include "VectorMath.h"
```
It is better to include again. That will insulate this file from compile errors if headers ever change what they include.